src/test/Tests/Integration.hs

```diff
@@ -145,6 +149,8 @@ integrationTests = testGroup "Solidity Integration Testing"
       , ("unexpected corpus count ",               countCorpus 5)]
   ,  testContract "basic/payable.sol"     Nothing
       [ ("echidna_payable failed",                 solved      "echidna_payable") ]
+  ,  testContract "basic/killed.sol"      Nothing
+      [ ("echidna_still_alive failed",                 solved      "echidna_still_alive") ]
```

Comment:
seems like this is not correctly aligned

Reply:
Fixed.

Reply:
Fixed.